src/lib/pickles/plonk_checks/scalars.ml

```diff
@@ -252,6 +318,324 @@ module Tick : S = struct
+            * ( vanishes_on_last_4_rows
+              * ( cell (var (LookupAggreg, Next))
+                  * ( if_feature
+                        ( LookupsPerRow 0
+                        , (fun () ->
+                            gamma
+                            * ( beta
+                              + field
+                                  "0x0000000000000000000000000000000000000000000000000000000000000001"
+                              )
+                            + cell (var (LookupSorted 0, Curr))
+                            + (beta * cell (var (LookupSorted 0, Next))) )
+                        , fun () ->
+                            field
+                              "0x0000000000000000000000000000000000000000000000000000000000000001"
+                        )
+                    * if_feature
+                        ( LookupsPerRow 1
+                        , (fun () ->
+                            gamma
+                            * ( beta
+                              + field
+                                  "0x0000000000000000000000000000000000000000000000000000000000000001"
+                              )
+                            + cell (var (LookupSorted 1, Next))
+                            + (beta * cell (var (LookupSorted 1, Curr))) )
+                        , fun () ->
+                            field
+                              "0x0000000000000000000000000000000000000000000000000000000000000001"
+                        )
+                    * if_feature
+                        ( LookupsPerRow 2
+                        , (fun () ->
+                            gamma
+                            * ( beta
+                              + field
+                                  "0x0000000000000000000000000000000000000000000000000000000000000001"
+                              )
+                            + cell (var (LookupSorted 2, Curr))
+                            + (beta * cell (var (LookupSorted 2, Next))) )
+                        , fun () ->
+                            field
+                              "0x0000000000000000000000000000000000000000000000000000000000000001"
+                        )
+                    * if_feature
+                        ( LookupsPerRow 3
+                        , (fun () ->
+                            gamma
+                            * ( beta
+                              + field
+                                  "0x0000000000000000000000000000000000000000000000000000000000000001"
+                              )
+                            + cell (var (LookupSorted 3, Next))
+                            + (beta * cell (var (LookupSorted 3, Curr))) )
+                        , fun () ->
+                            field
+                              "0x0000000000000000000000000000000000000000000000000000000000000001"
+                        )
+                    * if_feature
+                        ( LookupsPerRow 4
```

Comment:
Oh, so the `max_lookups_per_row` in proof-systems refers indeed to a maximum bound... So far we only have patterns that perform 4, 3, and 2 lookups per row. Why do we need to consider the 0 and 1 case as well here? For correctness? Or just in case we add other patterns in the future?